tests/e2e/run-scanner-v4.bats

```diff
@@ -70,6 +72,202 @@ teardown() {
     run remove_existing_stackrox_resources
 }
 
+# We are using our own deploy function, because we want to have the flexibility to patch down resources
+# after deployment. Without this we are only able to special-case local deployments and CI deployments,
+# but not, for example, manual testing on Infra.
+#
+# shellcheck disable=SC2120
+_deploy_stackrox() {
+    local tls_client_certs=${1:-}
+    local central_namespace=${2:-stackrox}
+    local sensor_namespace=${3:-stackrox}
+
+    deploy_stackrox_operator
```

Comment:
```suggestion
    setup_podsecuritypolicies_config

    deploy_stackrox_operator
```

This line is missing here and is present in the original `deploy_stackrox` which means that PSPs will not be tested in the OCP 4.11 job. I don't think it's very important, so feel free to resolve this.

Reply:
I think it's ok, but I will consider this change for a follow-up PR>